source/layouts/topics.haml

```diff
@@ -13,8 +13,8 @@
 
   .container
     .row
-      .col-xs-3
-        = partial 'support_sidebar'
+      .col-xs-2.hidden-xs
+        / = partial 'support_sidebar'
```

Comment:
If they get to that page, wouldn't we prefer they scan it first, then hit us up for help?